worlds/witness/items.py

```diff
@@ -57,9 +59,17 @@ def __init__(self):
 
             item_tab[item[0]] = ItemData(158000 + item[1], True, False)
 
+            self.ITEM_NAME_GROUPS.setdefault("Symbols", set()).add(item[0])
+
         for item in StaticWitnessLogic.ALL_DOOR_ITEMS:
             item_tab[item[0]] = ItemData(158000 + item[1], True, False)
 
+            if item[1] in range(1500, 1512):
+                self.ITEM_NAME_GROUPS.setdefault("Lasers", set()).add(item[0])
```

Comment:
I feel like it would be better if you would use some constants there, or at least add a comment. It took me some time to understand the code.
Also `range(1500, 1512)` includes 1511, but i haven't found a an item with 1511 in WitnessItems.txt